src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationServiceTests.cs

```diff
@@ -32,12 +34,624 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
 {
     public class PublicationServiceTests
     {
+        [Fact]
+        public async Task ListPublications_CanViewAllPublications()
+        {
+            var topic = new Topic
+            {
+                Title = "Topic title",
+                Theme = new Theme
+                {
+                    Title = "Theme title",
+                },
+            };
+
+            var publication = new Publication
+            {
+                Title = "Test Publication",
+                Summary = "Test summary",
+                Slug = "test-slug",
+                Topic = topic,
+                Contact = new Contact
+                {
+                    ContactName = "contact name",
+                    ContactTelNo = "1234",
+                    TeamName = "team name",
+                    TeamEmail = "team@email",
+                },
+                SupersededBy = null,
+            };
+
+            var userService = new Mock<IUserService>(Strict);
+
+            userService.Setup(s => s.GetUserId()).Returns(new Guid());
+            userService.Setup(s => s.MatchesPolicy(CanAccessSystem)).ReturnsAsync(true);
+            userService.Setup(s => s.MatchesPolicy(CanViewAllPublications)).ReturnsAsync(true);
+
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.Publications.AddRangeAsync(publication);
```

Comment:
This test case isn't very thorough as we haven't saved another publication with a different topic to prove that we're actually filtering down the publications based on the `topicId`.